proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -130,6 +130,8 @@ message WorkflowExecutionInfo {
     // If continued-as-new, or retried, or cron, holds the new run id.
     string new_execution_run_id = 61;
     temporal.server.api.clock.v1.ShardClock parent_clock = 62;
+    // version of child execution initiated event in parent workflow
+    int64 initiated_version = 63; 
```

Comment:
rename `int64 initiated_id = 6;` -> `int64 parent_initiated_id = 6;`
and this `int64 initiated_version = 63; ` -> `int64 parent_initiated_version = 63; `

?

Reply:
Update the naming in WorkflowExecutionInfo and RecordChildExecutionCompleted.